src/add-remove.test.js

```diff
@@ -0,0 +1,20 @@
+const myModules = require('./add-remove.js');
+
+let { listContainer } = myModules;
+const { addTask, deleteTask } = myModules;
+describe('my tasks list', () => {
+  // Test for adding tasks to the array as an object
+  test('test: add 3 tasks', () => {
+    addTask('Work');
+    addTask('Code');
+    addTask('Sleep');
+    expect(listContainer[1].description).toBe('Code');
+    expect(listContainer).toHaveLength(3);
+  });
+
+  // Test for removing tasks from the container
+  test('test: delete tasks', () => {
+    listContainer = deleteTask(2);
+    expect(listContainer).toHaveLength(2);
+  });
+});
```

Comment:
- [ ] Good work importing the functions and testing them, but as requested by the previews reviewer please use mock for functions that used localStorage or DOM

Reply:
Thank you for the reviews, they are really helpful. But the functions we tested doesn't have any localstorage or DOM, so that's why we didn't use mock for them.